plugins/single_plugins/resize.cpp

```diff
@@ -308,6 +311,16 @@ class wayfire_resize : public wf::per_output_plugin_instance_t, public wf::point
         min_size = wf::expand_dimensions_by_margins(min_size,
             view->toplevel()->pending().margins);
 
+        if (user_min_width)
+        {
+            min_size.width = user_min_width;
```

Comment:
```suggestion
            min_size.width = std::max(user_min_width, min_size.width);
```

Idea: maybe the user says min size 50pixels, but app says min size 200 pixels, we can't resize lower than 200 pixels.